doc/reference/reference_lua/config.rst

```diff
@@ -395,12 +395,16 @@ The ``config.storage`` API allows you to interact with a Tarantool-based :ref:`c
 
 .. _config_storage_api_reference_put:
 
-.. function:: put(path, value)
+.. function:: put(path, value, opts)
 
     Put a value by the specified path.
 
     :param string path: a path to put the value by
     :param string value: a value to put
+    :param table opts:   a table containing the following optional fields:
+
+                            *   ``ttl`` (since :doc:`3.2.0 </release/3.2.0>`, default: unset): time-to-live in seconds, if nil or not set the key won't expire, may issue an error if set on config.storage running old schema, see :ref:`config.storage.info <config_storage_api_reference_info>`.
```

Comment:
AFAIR, we discussed a lot the following corner cases:

* If a key exists and has no TTL, what `put` of this key with TTL do?
* Vice versa: if the key exists and has TTL, what `put` of this key without TTL do?
* Next, if a key has TTL that expires at time T and we attempt to `put` this key with a TTL that expires at time T1 < T, whether the expire occurs on T1 or T? How about T2 > T?

I would like if the API documentation would describe a behavior in these scenarios.